materials/knitr.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Knitr
+layout: default
+---
+
+## Literate programming
+
+* Combine text, code, figures, tables, etc.
+* Write papers without having to remember to swap figures
+* Automatically generate reports
+
+
+## Getting started
+
+* File -> New File -> R Markdown
+* Enter a title and author(s)
+* Shows a basic stub of a document
+* Press `Knit PDF` to create a pdf from it
+    * 1st time may ask you to install some packages
+* Runs the code in the code chunks and prints their output along with the
+  markdown formatted text
+
+
+## Markdown
+
+* Basic approach to formatting text
+* Let's you do
+    * `*italics*`
+	* `**bold**`
+	* `[links](http://google.com)`
+	* Lists
+	    * `*`
+		* `1.`
+	* `# Header`
+* Easy to read for humans
+* Easy to convert into other things for computers
+* Common on lots of websites
+* Used to create all of the exercises and lectures in this course
+* Github will automatically render it
+    * https://github.com/ethanwhite/CV/blob/master/CV.md
+
+
+## Example
+
+	Explore patterns in population dynamics at Portal.
+
+	## Required Libraries 
+	```{r}
```

Comment:
I think this will be formatted to a code block by Jekyll. Did you want to keep the backticks visible?